tests/lax_numpy_test.py

```diff
@@ -1505,6 +1505,23 @@ def testDiagFlat(self, shape, dtype, k):
     self._CheckAgainstNumpy(np_fun, jnp_fun, args_maker, check_dtypes=True)
     self._CompileAndCheck(jnp_fun, args_maker, check_dtypes=True)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_shape1={}_shape2={}_{}".format(
```

Comment:
Please use `jtu.format_shape_dtype_string` for both shapes (repeated dtype is fine), and make sure to use `a1_shape`/`a2_shape` in the test name